lib/at_ex.ex

```diff
@@ -7,9 +7,14 @@ defmodule AtEx do
   - Consuming incoming events that have been parsed
   - Building valid responses
   """
-  alias AtEx.{
-    Gateway.Airtime
+  alias AtEx.Gateway.{
+    Airtime,
+    Application,
+    Sms
   }
 
   defdelegate send_airtime(map), to: Airtime
+  defdelegate get_data(), to: Application
```

Comment:
@kamalogudah  perhaps call this `get_application_data`, also I think you many want to start documenting this functions so that users can know how to use them